keras/backend/tensorflow/trackable.py

```diff
@@ -0,0 +1,58 @@
+import tensorflow as tf
+
+from keras.utils import tracking
+
+
+class KerasAutoTrackable(tf.__internal__.tracking.AutoTrackable):
```

Comment:
If the tracking is done strictly by hand, is it better to subclass from `Trackable` to minimize interference from TF?